### DISCUSSION TITLE
Distribute .ipa for iOS and support AltStore

### DISCUSSION THREAD

**Discussion by Gigas002** [ppy/osu#18988] [Ideas] [answered]
[AltStore](https://github.com/rileytestut/AltStore) had released the 1.5 version recently, which adds an ability for apps to be added from custom sources (for example, see [Yattee's](https://github.com/yattee/yattee) - the alternative open-source frontend for youtube [json](https://repos.yattee.stream/alt/apps.json) for this.
It would be an awesome addition for players on iOS, since testflight program was filled up ages ago and no opportunity for participate for several years already. I'm pretty sure it's not hard to slightly change iOS CI to push built `.ipa` assets to the github releases.

**Comment by integerrr** [comment, marked as answer]
#9795
Edit: the apps will eventually be on the respective app stores for the OSes so even one less reason to distribute `.ipa`s

**Reply by Gigas002** [reply]
Yeah, I've seen this. What's the reason behind that decision?

**Reply by peppy** [reply]
I do not want people to use such methods to load the app. We will be releasing properly in the near future so stay tuned.

**Reply by ferologics** [reply]
still no release 😅

**Reply by peppy** [reply]
your eyes are astute.

**Reply by ferologics** [reply]
@peppy wanted to try it out on iOS today, but TF beta is full, and don't see anything in Altstore ... I guess I'm left with trying to compile it myself with Xcode?

**Reply by Gigas002** [reply]
I haven't been using Altstore for years, but if I remember correctly you can create your own feed using some `.json` file and a github repo. The compiled `osu.iOS.ipa` is available on releases page, though again - I have never tested it, but I think it's doable
